includes/gateway/abstract-omise-payment-base-card.php

```diff
@@ -63,19 +63,23 @@ private function prepareChargeData($order_id, $order, $omise_customer_id, $card_
 		$data = [
 			'amount' => Omise_Money::to_subunit($order->get_total(), $currency),
 			'currency' => $currency,
-			'description' => apply_filters(
```

Comment:
I Just would like to understand why are we removing `apply_filters` here?

Reply:
Because this apply_filter is doing nothing. There is no where in the code where `omise_charge_params_description` filter is added. Maybe it was there before and it was removed later. I don't know.

Even with `apply_filters`, we are receiving `WooCommerce Order id <order-id>` as charge description in the Opn dashboard.